### PR TITLE
Event list paginated

### PR DESCRIPTION
**Pull request by diegofsousa**
"Event list" paginated.
15 events per page.

Image:
___________________________________________________
paginate
___________________________________________________

What do you think?
Thank's :)

### CODE REVIEW

File: deck/models.py [hunk 1]
```diff
@@ -349,6 +350,28 @@ def get_not_approved_schedule(self):
             .order_by('-is_approved', '-votes__rate__sum')
         return not_approved_schedule
 
+    def get_range_page(self, page_number):
```
Comment (luanfonceca) [inline deck/models.py]: I think that this method can be moved to `deck_tags.py`. You created a method on the Event model, that does not use the event instance and then when you need to call that method you are creating an unnecessary object just to call it.

**Comment by luanfonceca** [comment]
Hi, i merged your pull request and did some changes on the code.
I need to implement the Search with pagination feature too.

You can see all changes on the commit 8e0aaa9feed48247366ab3047cb3fe3765a279e2

**Comment by diegofsousa** [comment]
Thank you, this is my first PR in open-source projects.
I'm sorry for something i did wrong.

**Comment by luanfonceca** [comment]
Haha, its okay. If you want to take another issue, come talk to me and we work on that together in portuguese o/

**Pull request closed without merging**